app/services/webhook/webhook.go

```diff
@@ -0,0 +1,173 @@
+package webhook
+
+import (
+	"context"
+	"fmt"
+	"github.com/getfider/fider/app/models/cmd"
+	"github.com/getfider/fider/app/models/dto"
+	"github.com/getfider/fider/app/models/entity"
+	"github.com/getfider/fider/app/models/query"
+	"github.com/getfider/fider/app/pkg/bus"
+	"github.com/getfider/fider/app/pkg/log"
+	"github.com/getfider/fider/app/pkg/tpl"
+	"net/http"
+	"strings"
+)
+
+func init() {
+	bus.Register(Service{})
+}
+
+type Service struct{}
+
+func (s Service) Name() string {
+	return "Webhook"
+}
+
+func (s Service) Category() string {
+	return "hook"
+}
+
+func (s Service) Enabled() bool {
+	return true
+}
+
+func (s Service) Init() {
+	bus.AddHandler(triggerWebhook)
+	bus.AddHandler(triggerWebhooksByType)
+	bus.AddHandler(previewWebhook)
+	bus.AddHandler(getWebhookProps)
+}
+
+func triggerWebhook(ctx context.Context, c *cmd.TriggerWebhook) error {
```

Comment:
It seems like this is only used for testing, maybe we rename it to "testWebhook" or something similar? looking at triggerWebhook it gave me the impression this was the real trigger.

by using a different name here, maybe the one belo could then be just "cmd.TriggerWebhooks"

Reply:
By testing you mean the "Trigger" button in the admin panel I suppose? Maybe `TestWebhook` would be confusing with unit tests, so `DummyTriggerWebhook`. Even if I don't feel very convinced by this name... Because in fact this is really triggering the Webhook, just with dummy props, not only testing the templates and URL validity for example.
I agree about the rename for the other command

Reply:
Which makes me think that maybe the button should be "Test", just like the OAUth one? We test a webhook by triggering it with dummy props, but the action itself is still "Test" in my view.

Reply:
I renamed those elements but kept the `WebhookTriggerResult` because it would be used by the hypothetical execution logging system